src/SockPerf.cpp

```diff
@@ -1564,11 +1568,11 @@ static int proc_mode_server( int id, int argc, const char **argv )
 		printf("\n");
 		printf("Usage: " MODULE_NAME " %s [options] [args]...\n", sockperf_modes[id].name);
 		printf(" " MODULE_NAME " %s\n", sockperf_modes[id].name);
-		printf(" " MODULE_NAME " %s [-i ip] [-p port] [--rx-mc-if ip] [--tx-mc-if ip]\n", sockperf_modes[id].name);
+		printf(" " MODULE_NAME " %s [-i ip] [-p port] [--rx-mc-if ip] [--tx-mc-if ip] [--mc-source-ip]\n", sockperf_modes[id].name);
 #ifndef WIN32
-		printf(" " MODULE_NAME " %s -f file [-F s/p/e] [--rx-mc-if ip] [--tx-mc-if ip]\n", sockperf_modes[id].name);
+		printf(" " MODULE_NAME " %s -f file [-F s/p/e] [--rx-mc-if ip] [--tx-mc-if ip] [--mc-source-ip]\n", sockperf_modes[id].name);
 #else
-		printf(" " MODULE_NAME " %s -f file [-F s] [--rx-mc-if ip] [--tx-mc-if ip]\n", sockperf_modes[id].name);
+		printf(" " MODULE_NAME " %s -f file [-F s] [--rx-mc-if ip] [--tx-mc-if ip] [--mc-source-ip]\n", sockperf_modes[id].name);
 #endif
```

Comment:
+AlexR


From: ashanably [mailto:notifications@github.com]
Sent: Tuesday, August 02, 2016 14:57
To: Mellanox/sockperf <sockperf@noreply.github.com>
Cc: Avner BenHanoch <avnerbh@gmail.com>; Comment <comment@noreply.github.com>
Subject: Re: [Mellanox/sockperf] Support multicat source filtering (#74)


In src/SockPerf.cpp<https://github.com/Mellanox/sockperf/pull/74#discussion_r73141294>:

>  #else

> -           printf(" " MODULE_NAME " %s -f file [-F s] [--rx-mc-if ip] [--tx-mc-if ip]\n", sockperf_modes[id].name);

> +           printf(" " MODULE_NAME " %s -f file [-F s] [--rx-mc-if ip] [--tx-mc-if ip] [--mc-source-ip]\n", sockperf_modes[id].name);

>  #endif

we can change it but its rule is somehow different than rx-mc-if and tx-mc-if so I didn't use the same format.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/Mellanox/sockperf/pull/74/files/0663b35853a70c465ef2d58db19fa9a792ea8561#r73141294>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ADlNzHGUldFg2uYwASpmvNXNbFpGoDyAks5qbzBxgaJpZM4JaYCw>.